binderhub/builder.py

```diff
@@ -408,33 +426,52 @@ async def get(self, provider_prefix, _unescaped_spec):
             else:
                 image_found = True
 
-        if image_found:
+        build_only_outcome = self._get_build_only_outcome()
```

Comment:
Same here, let's call it `build_only`, which I think is clearer than adding `outcome`